src/main/resources/templates/logback/logback.tmpl

```diff
@@ -1,14 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-	<property name="LOG_FILE" value="logs/application.log"/>
+    <property name="LOG_FILE" value="logs/application.log"/>
```

Comment:
I'm pretty sure these properties need to be defined _after_ defaults.xml is pulled in, otherwise it overrides them.  see https://github.com/blackbaud/bluemoon-dojo/blob/master/src/main/resources/logback.xml